modules/meilisearch/go.mod

```diff
@@ -0,0 +1,63 @@
+module github.com/testcontainers/testcontainers-go/modules/meilisearch
+
+go 1.23.1
+
+require (
+	github.com/docker/docker v27.3.1+incompatible
+	github.com/docker/go-units v0.5.0
```

Comment:
chore: please run go mod tidy so that those two imports are moved to the indirect dependencies